### PR TITLE
[NOD-1163] Combine seperated flows into single packages

### PR DESCRIPTION
**Pull request by svarogg**

### CODE REVIEW

File: protocol/handshake.go [hunk 1]
```diff
@@ -4,13 +4,13 @@ import (
 	"sync"
 	"sync/atomic"
 
+	handshake2 "github.com/kaspanet/kaspad/protocol/flows/handshake"
```
Comment (someone235) [inline protocol/handshake.go:7]: handshake2 ?

File: logger/logger.go [hunk 2]
```diff
@@ -51,11 +51,11 @@ var (
 	grpcLog = BackendLog.Logger("GRPC")
 	p2psLog = BackendLog.Logger("P2PS")
 	ntarLog = BackendLog.Logger("NTAR")
-	blkrLog = BackendLog.Logger("BLKR")
-	gbrlLog = BackendLog.Logger("GBRL")
+	brflLog = BackendLog.Logger("BRFL")
 	blprLog = BackendLog.Logger("BLPR")
 	dnssLog = BackendLog.Logger("DNSS")
 	snvrLog = BackendLog.Logger("SNVR")
+	hsflLog = BackendLog.Logger("HSFL")
```
Comment (stasatdaglabs) [inline logger/logger.go:58]: Please use one logger for everything inside the protocol package.

File: protocol/handshake.go (hunk 1 above)
Comment (stasatdaglabs) [inline protocol/handshake.go:7]: Is there any reason for handshake2? In not, please remove it.

File: protocol/flows/handshake/handshake.go [hunk 3]
```diff
@@ -29,9 +29,9 @@ func handshake(router *routerpkg.Router, netAdapter *netadapter.NetAdapter, peer
 		panic(err)
 	}
 
-	// For the handshake to finish, we need to get from the other node
+	// For the addHandshakeFlow to finish, we need to get from the other node
```
Comment (someone235) [inline protocol/flows/handshake/handshake.go:32]: Wrong rename

File: protocol/flows/handshake/handshake.go [hunk 4]
```diff
@@ -29,9 +29,9 @@ func handshake(router *routerpkg.Router, netAdapter *netadapter.NetAdapter, peer
...
 	// a version and verack messages, so we increase the wait group by 2
-	// and block the handshake with wg.Wait().
+	// and block the addHandshakeFlow with wg.Wait().
```
Comment (someone235) [inline protocol/flows/handshake/handshake.go:34]: Wrong rename

Comment (stasatdaglabs) [inline protocol/flows/handshake/handshake.go:34]: Refactor error.

**Pull request merged**